.github/workflows/core.yml

```
@@ -288,8 +288,8 @@ jobs:
       PYTEST_TIMEOUT: ${{ matrix.asan == 'asan' && 600 || 400 }}
       ACTIONS_DO_UI_TEST: ${{ matrix.coins == 'universal' && matrix.asan == 'noasan' }}
       TEST_LANG: ${{ matrix.lang }}
-      TESTOPTS: "--durations 10 --session-timeout 1800"  # 30m pytest global timeout
-    timeout-minutes: 40
+      TESTOPTS: "--durations 10 --session-timeout ${{ env.TREZOR_MODEL == 'T3W1' && '2400' || '1800' }}"  # pytest global timeout
```

actionlint (1.7.11)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)